app/components/Footer/Footer.tsx

```diff
@@ -12,7 +12,7 @@ import { BsTiktok } from 'react-icons/bs'
 
 const Footer = () => {
 	return (
-		<footer className="flex justify-between md:justify-end md:gap-16 mt-20 mb-4">
+		<footer className="flex justify-center md:gap-16 mt-20 mb-4">
```

Comment:
Add `justify-center` only for `md:` since for mobile I still want it to have `justify-between`